src/Core/Layer/LayerUpdateState.js

```diff
@@ -31,6 +31,9 @@ LayerUpdateState.prototype.canTryUpdate = function canTryUpdate(timestamp) {
         }
         case UPDATE_STATE.ERROR:
         default: {
+            if (this.failureParams) {
```

Comment:
Why do you want to retry without a pause here?

Reply:
I don't think there is a delay in trying to fetch a new node with a different level

Reply:
yes, i updated the PR
if two last Failure params aren't equivalent we can try without delay

Reply:
You added some complicated code to do this, but IMHO you missed my point.

The code https://github.com/iTowns/itowns/pull/618/files#diff-ad71c3a479d08240c3053171dd65d576R285 can't distinguish between a broken data server (in this case retrying with a lower level can make sense), a network error (wifi connection lost, in which case skipping the error delay is causing more harm than good).

So basically, I'd prefer to be on the safe side, and always use the pause delay in case of error.

Btw, did you try to turn off your internet connection while using itowns to see what happens before/after this PR?

Reply:
In this PR in offline, itowns retries with delay we find the current mechanism.
because there are several errors on the same fail parameters.

if we respect the pause delay to try again with new parameters
we can wait for more than 10 seconds the best texture